### ISSUE TITLE
error when yarn run webpack

### ISSUE THREAD

**Issue by wkgreat** [openlayers/openlayers#15975]
after yarn run webpack, got an error
```
WARNING in ./node_modules/ol/structs/RBush.js
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/structs/RBush.js
    Used by 1 module(s), i. e.
    javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/source/Vector.js
* javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/structs/rbush.js
    Used by 2 module(s), i. e.
    javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/structs/RBush.js
 @ ./node_modules/ol/source/Vector.js 10:0-40 258:48-53 264:35-40
 @ ./node_modules/ol/layer/Graticule.js 14:0-47 470:10-22
 @ ./node_modules/ol/index.js 9:0-58 9:0-58
 @ ./spacture/ol/olMap.ts 17:10-23
 @ ./spacture/ol/map.tsx 17:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:34-81
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74
```

**To Reproduce**
```
yarn add ol
yarn run webpack --config webpack.config.js --progress --watch
```

**Comment by wkgreat** [comment]
additional error
```
WARNING in ./node_modules/ol/structs/RBush.js
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/structs/RBush.js
    Used by 3 module(s), i. e.
    javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/source/Vector.js
* javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/structs/rbush.js
    Used by 2 module(s), i. e.
    javascript/esm|/workspaces/pgadmin4/web/node_modules/ol/structs/RBush.js
 @ ./node_modules/ol/source/Vector.js 24:0-40 252:53-58 257:40-45
 @ ./node_modules/ol/source.js 30:0-55 30:0-55
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/GeoTIFF.js 77:25-29
export 'Pool' (imported as 'Pool') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/GeoTIFF.js 189:18-30
export 'fromBlob' (imported as 'tiffFromBlob') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/GeoTIFF.js 192:18-30
export 'fromUrls' (imported as 'tiffFromUrls') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/GeoTIFF.js 195:18-29
export 'fromUrl' (imported as 'tiffFromUrl') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/geotiff.js 77:25-29
export 'Pool' (imported as 'Pool') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source/GeoTIFF.js 21:0-108 77:25-29 189:18-30 192:18-30 195:18-29
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/geotiff.js 189:18-30
export 'fromBlob' (imported as 'tiffFromBlob') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source/GeoTIFF.js 21:0-108 77:25-29 189:18-30 192:18-30 195:18-29
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/geotiff.js 192:18-30
export 'fromUrls' (imported as 'tiffFromUrls') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source/GeoTIFF.js 21:0-108 77:25-29 189:18-30 192:18-30 195:18-29
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74

ERROR in ./node_modules/ol/source/geotiff.js 195:18-29
export 'fromUrl' (imported as 'tiffFromUrl') was not found in 'geotiff' (possible exports: default)
 @ ./node_modules/ol/source/GeoTIFF.js 21:0-108 77:25-29 189:18-30 192:18-30 195:18-29
 @ ./node_modules/ol/source.js 10:0-57 10:0-57
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/OLMAP.jsx 11:14-34
 @ ./pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx 82:36-54
 @ ./pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx 56:17-48
 @ ./pgadmin/tools/sqleditor/static/js/SQLEditorModule.js 57:49-91
 @ ./pgadmin/tools/sqleditor/static/js/index.js 6:46-74
```

**Comment by wkgreat** [comment]
```
(venv) npm ls ol
web@ /workspaces/pgadmin4/web
└── ol@6.15.1

(venv) 
```
I have tried difference ol version, like 6.15.1, 9.2.4. I got same error. this error seems like not dependent on version

**Comment by wkgreat** [comment]
I also tried renove node_module, it do not work. I confirmed there are only one node_module.

**Comment by wkgreat** [comment]
there's only one GeoTIFF.js
```
(venv) cd node_modules/ol/source/
(venv) ls
BingMaps.d.ts             ImageArcGISRest.d.ts.map  OSM.js                    TileEventType.js.map      VectorTile.d.ts
BingMaps.d.ts.map         ImageArcGISRest.js        OSM.js.map                TileImage.d.ts            VectorTile.d.ts.map
BingMaps.js               ImageArcGISRest.js.map    Raster.d.ts               TileImage.d.ts.map        VectorTile.js
BingMaps.js.map           ImageCanvas.d.ts          Raster.d.ts.map           TileImage.js              VectorTile.js.map
CartoDB.d.ts              ImageCanvas.d.ts.map      Raster.js                 TileImage.js.map          WMTS.d.ts
CartoDB.d.ts.map          ImageCanvas.js            Raster.js.map             TileJSON.d.ts             WMTS.d.ts.map
CartoDB.js                ImageCanvas.js.map        Source.d.ts               TileJSON.d.ts.map         WMTS.js
CartoDB.js.map            ImageMapGuide.d.ts        Source.d.ts.map           TileJSON.js               WMTS.js.map
Cluster.d.ts              ImageMapGuide.d.ts.map    Source.js                 TileJSON.js.map           XYZ.d.ts
Cluster.d.ts.map          ImageMapGuide.js          Source.js.map             TileWMS.d.ts              XYZ.d.ts.map
Cluster.js                ImageMapGuide.js.map      Stamen.d.ts               TileWMS.d.ts.map          XYZ.js
Cluster.js.map            ImageStatic.d.ts          Stamen.d.ts.map           TileWMS.js                XYZ.js.map
DataTile.d.ts             ImageStatic.d.ts.map      Stamen.js                 TileWMS.js.map            Zoomify.d.ts
DataTile.d.ts.map         ImageStatic.js            Stamen.js.map             UTFGrid.d.ts              Zoomify.d.ts.map
DataTile.js               ImageStatic.js.map        Tile.d.ts                 UTFGrid.d.ts.map          Zoomify.js
DataTile.js.map           ImageWMS.d.ts             Tile.d.ts.map             UTFGrid.js                Zoomify.js.map
GeoTIFF.d.ts              ImageWMS.d.ts.map         Tile.js                   UTFGrid.js.map            common.d.ts
GeoTIFF.d.ts.map          ImageWMS.js               Tile.js.map               UrlTile.d.ts              common.d.ts.map
GeoTIFF.js                ImageWMS.js.map           TileArcGISRest.d.ts       UrlTile.d.ts.map          common.js
GeoTIFF.js.map            OGCMapTile.d.ts           TileArcGISRest.d.ts.map   UrlTile.js                common.js.map
IIIF.d.ts                 OGCMapTile.d.ts.map       TileArcGISRest.js         UrlTile.js.map            ogcTileUtil.d.ts
IIIF.d.ts.map             OGCMapTile.js             TileArcGISRest.js.map     Vector.d.ts               ogcTileUtil.d.ts.map
IIIF.js                   OGCMapTile.js.map         TileDebug.d.ts            Vector.d.ts.map           ogcTileUtil.js
IIIF.js.map               OGCVectorTile.d.ts        TileDebug.d.ts.map        Vector.js                 ogcTileUtil.js.map
Image.d.ts                OGCVectorTile.d.ts.map    TileDebug.js              Vector.js.map             wms.d.ts
Image.d.ts.map            OGCVectorTile.js          TileDebug.js.map          VectorEventType.d.ts      wms.d.ts.map
Image.js                  OGCVectorTile.js.map      TileEventType.d.ts        VectorEventType.d.ts.map  wms.js
Image.js.map              OSM.d.ts                  TileEventType.d.ts.map    VectorEventType.js        wms.js.map
ImageArcGISRest.d.ts      OSM.d.ts.map              TileEventType.js          VectorEventType.js.map
```

**Comment by ahocevar** [comment]
Ok, then I'm out of ideas. Must be something with your specific build setup, but not an OpenLayers issue. At least I cannot reproduce it.

**Comment by wkgreat** [comment]
ok, let me try again...

**Comment by wkgreat** [comment]
got a nasty workaround, 
replace the line in ./node_modules/ol/source/GeoTIFF.js
```
import { Pool, fromBlob as tiffFromBlob, fromUrl as tiffFromUrl, fromUrls as tiffFromUrls, } from 'geotiff';
```
to
```
import { Pool, fromBlob as tiffFromBlob, fromUrl as tiffFromUrl, fromUrls as tiffFromUrls, } from '/workspaces/pgadmin4/web/node_modules/geotiff/dist-module/geotiff';
```

**Comment by ahocevar** [comment]
Looks like a webpack issue. Maybe you want to report the problem there.
